pkg/resources/management.cattle.io/v3/cluster/validator_test.go

```diff
@@ -522,9 +522,6 @@ func TestAdmit(t *testing.T) {
 					assert.Equal(t, tt.expectedReason, res.Result.Reason)
 				}
 			}
-			if tt.expectContainWarning {
```

Comment:
Looks like it's indeed safe to just remove this check altogether since the only tests with `expectContainWarning` set to `true` are for the exact warning we're removing (please double-check). However, we also need then to FULLY remove `expectContainWarning` as it serves no purpose anymore.